js/image/src/Webcam.svelte

```diff
@@ -23,7 +23,11 @@
 			| string;
 		error: string;
 		start_recording: undefined;
-		stop_recording: undefined;
+		stop_recording: {
+			data: FileReader["result"];
+			name: string;
+			is_example?: boolean;
+		};
```

Comment:
We can update this type (see later comment for context).

```suggestion
		stop_recording: undefined;
```